src/server.rs

```diff
@@ -115,9 +115,11 @@ impl Service for Server {
         // from http://jaketrent.com/post/https-redirect-node-heroku/
         if let Some(raw) = req.headers().get_raw("x-forwarded-proto") {
             if raw != &b"https"[..] {
+                let host: &Host = req.headers().get().unwrap();
```

Comment:
Hmmm I tried both refactors and got some errors from the compiler.

`let host = req.headers().get().unwrap::<&Host>();`:

```rust
   Compiling sparkles v0.1.0 (file:///Users/hipsterbrown/Projects/sparkles)
error[E0035]: does not take type parameters
   --> src/server.rs:118:48
    |
118 |                 let host = req.headers().get().unwrap::<&Host>();
    |                                                ^^^^^^ called with unneeded type parameters

error: aborting due to previous error

error: Could not compile `sparkles`.

To learn more, run the command again with --verbose.

shell returned 101
```

-----

`let host = req.headers().get::<Option<&Host>>().unwrap();`

```rust
   Compiling sparkles v0.1.0 (file:///Users/hipsterbrown/Projects/sparkles)
error[E0277]: the trait bound `std::option::Option<&hyper::header::Host>: hyper::header::Header` is not satisfied
   --> src/server.rs:118:42
    |
118 |                 let host = req.headers().get::<Option<&Host>>().unwrap();
    |                                          ^^^ the trait `hyper::header::Header` is not implemented for `std::option::Option<&hyper::header::Host>`

error[E0277]: the trait bound `std::option::Option<&hyper::header::Host>: std::fmt::Display` is not satisfied
   --> src/server.rs:122:26
    |
122 |                          host,
    |                          ^^^^ the trait `std::fmt::Display` is not implemented for `std::option::Option<&hyper::header::Host>`
    |
    = note: `std::option::Option<&hyper::header::Host>` cannot be formatted with the default formatter; try using `:?` instead if you are using a format string
    = note: required because of the requirements on the impl of `std::fmt::Display` for `&std::option::Option<&hyper::header::Host>`
    = note: required by `std::fmt::Display::fmt`

error: aborting due to 2 previous errors

error: Could not compile `sparkles`.

To learn more, run the command again with --verbose.

shell returned 101
```

---

I'm not sure how to fix either of those issues.